src/lib/registration/storage.rb

```diff
@@ -131,14 +131,18 @@ def export
       end
 
       def import(settings)
+        Yast.import "AutoinstConfig"
+
         reset
 
+        product = Yast::AutoinstConfig.selected_product.short_name
```

Comment:
Um, does this work also in AY config mode in installed system?

Reply:
why not? ```selected_product``` just tries to identify a base product according to the information in the profile.

Reply:
And when the profile does not define the product (yet)?

Reply:
Then we get a P1: https://bugzilla.suse.com/show_bug.cgi?id=1075723

Reply:
I mean, this method is also called when AY is not used at all.

Reply:
@mchf UGH, does this mean that this code was supposed to be moved to scc_auto abut we forgot to delete it here??

Reply:
ups. will check

Reply:
Yes.